experiments/chat/chat/templates/chat/message.html

```diff
@@ -1 +1,2 @@
-<li id="message_{{ message.pk }}">{{ message.text }}</li>
+{% load turbo_streams %}
+<li id="{% stream_id message %}">{{ message.text }}</li>
```

Comment:
I would at Information here about the npm packages for people that don't like to use dcn.